### PR TITLE
#1646 CheckArchieveAndMoveToInbox ui test

### LINKED ISSUE #1646: UI test for archive and `move to inbox` feature
Let me merge this PR and create new issue for ui test for `archive` & `move to inbox`.

_Originally posted by @ioanmo226 in https://github.com/FlowCrypt/flowcrypt-ios/issues/1645#issuecomment-1137444856_

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR implemented CheckArchieveAndMoveToInbox ui test

close #1646 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    });
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      // First find email and archieve it
+      await MailFolderScreen.clickOnEmailBySubject(testMessageSubject);
+      await EmailScreen.clickArchiveButton();
```
Comment (ioanmo226) [inline appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts:30]: @sosnovsky For mocked api, it throws error when user wants to archive thread. It returns updateLabels it not a function error.
Could you have a look at this as you have a better understanding of mocked apis?
```
[0-13] url:POST:/gmail/v1/users/me/threads/17d48c645be70702/modify?prettyPrint=false TypeError: DATA[this.acct].messages[index].updateLabels is not a function
[0-13]     at GoogleData.updateMessageLabels (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-data.ts:267:41)
[0-13]     at /gmail/v1/users/me/threads/? (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-endpoints.ts:154:14)
[0-13]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[0-13]     at async LoggedApi.Api.handleReq (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/lib/api.ts:141:30)
[0-13] 1ms | 500 POST /gmail/v1/users/me/threads/17d48c645be70702/modify?prettyPrint=false | {"mockError":"DATA[this.acct].messages[index].updateLabels is not a function","mockStack":"TypeError: DATA[this.acct].messages[index].updateLabels is not a function\n    at GoogleData.updateMessageLabels (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-data.ts:267:41)\n    at /gmail/v1/users/me/threads/? (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-endpoints.ts:154:14)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async LoggedApi.Api.handleReq (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/lib/api.ts:141:30)","noFormatter":"if you wanted a different error format, subclass from HttpErr and add a formatter"}
[0-13] url:POST:/gmail/v1/users/me/threads/17d48c645be70702/modify?prettyPrint=false TypeError: DATA[this.acct].messages[index].updateLabels is not a function
[0-13]     at GoogleData.updateMessageLabels (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-data.ts:267:41)
[0-13]     at /gmail/v1/users/me/threads/? (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-endpoints.ts:154:14)
[0-13]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[0-13]     at async LoggedApi.Api.handleReq (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/lib/api.ts:141:30)

```

Comment (sosnovsky) [reply appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts]: @ioanmo226 yeah, I fixed it

Comment (ioanmo226) [reply appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts]: Great. Thanks

**Review by sosnovsky — DISMISSED** [review]

File: appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts (hunk 1 above)
Comment (ioanmo226) [reply appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts]: Hmmm.. ui tests fail now.
Maybe because of new changes?

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**